### PR TITLE
Living Wage KE

### PR DESCRIPTION
**Pull request by puritybirir**
##### What does this PR do?
- Add the correct data sets for Living Wage KE
- Create an introduction to Living Wage KE
- Make UI Changes ie font-size, font-type, colors and other small additions

##### Description of Task to be completed?
- Create a Living Wage Calculator for Living Wage KE

##### Any background context you want to provide?
- The previous Calculator was designed for South Africa

##### What are the relevant Pivotal Tracker stories?
- #164323157
- #164323542
- #164323521
- #164322752
- #164208259

##### Questions?
- None

##### Screenshots/Screencasts?
   **Introduction**
living wage
   **Calculator**
calculator

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 @import url(http://fonts.googleapis.com/css?family=Roboto+Slab:400,700,300);
+@import url(https://fonts.googleapis.com/css?family=Oswald:300,400);
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+@import url(https://fonts.googleapis.com/css?family=Nunito);
 
 body{
-    background: #ccc url('../images/furley_bg.png');
-    font-family: 'Roboto';
+    background: #fff;
+    font-family: 'Montseratt';
```
Comment (kilemensi) [inline css/styles.css]: Where do we include `Montseratt`?

File: css/styles.css [hunk 2]
```diff
@@ -63,16 +66,15 @@ h1, h2, h3, h4, h5, h6{
 .jumbotron p.lead{
 
     margin-bottom: 20px;
-
-    font-family: 'Roboto Slab';
+    font-family: 'sans-serif';
```
Comment (kilemensi) [inline css/styles.css]: Shouldn't it be one of the imported fonts?

File: css/styles.css [hunk 3]
```diff
@@ -380,3 +390,109 @@ h1, h2, h3, h4, h5, h6{
...
+    width: 40px;
+    height: 60px;
+  }
+  
+  .bounce {
+    -moz-animation: bounce 2s infinite;
+    -webkit-animation: bounce 2s infinite;
+    animation: bounce 2s infinite;
+  }
+
```
Comment (kilemensi) [inline css/styles.css]: New line

**Review by kilemensi — CHANGES_REQUESTED** [review]
Code wise seems cool @puritybirir, but:

 1. Does someone need to sit through the intro **every time** they visit the site? No skip, fast forward or even pause and rewind if you've missed a word?
 2. I think the arrow down should be clickable. Clicking it should take me to the calculator.
screenshot from 2019-03-08 12-37-41
 3. The share buttons seem broken to me.
screenshot from 2019-03-08 12-26-48

**Comment by puritybirir** [comment]
> Code wise seems cool @puritybirir, but:
> 
> 1. Does someone need to sit through the intro **every time** they visit the site? No skip, fast forward or even pause and rewind if you've missed a word?
> 2. I think the arrow down should be clickable. Clicking it should take me to the calculator.
>    screenshot from 2019-03-08 12-37-41
> 3. The share buttons seem broken to me.
>    screenshot from 2019-03-08 12-26-48

Resolved

File: index.html [hunk 4]
```diff
@@ -4,13 +4,14 @@
...
 
   <!-- Bootstrap -->
   <link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css" rel="stylesheet">
   <link href="css/bootstrap-select.min.css" rel="stylesheet">
   <link href="css/bootstrap-slider.css" rel="stylesheet">
   <link href="css/styles.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
   <!-- Font Awesome -->
   <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
```
Comment (kilemensi) [inline index.html]: Repeated font-awesome loading.

**Review by kilemensi — CHANGES_REQUESTED** [review]
@puritybirir:

 1. Ensure all resources are loaded via `//domain`, you're still mixing `http://` and `https://`,
 2. The KSH 8000 maximum seems a bit too low (both month and week will show red). But it's up to the data team I guess.
Screenshot from 2019-03-11 13-25-59


Other than that, I think it's good to go. :+1: .

-----------------------

PS: The broken Twitter and Facebook buttons was because of Firefox blocking [content](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Privacy/Tracking_Protection?utm_source=mozilla&utm_medium=firefox-console-errors&utm_campaign=default)

File: index.html [hunk 5]
```diff
@@ -55,732 +55,790 @@
...
-        <div class="row">
-          <div class="col-sm-12">
-            <p class="lead">
-              At this rate, you are paying the equivalent of <strong><span id="output-amount">R0</span></strong> per month.
-            </p>
-          </div>
+<div id='load'>
+    <div id="slideshow">
+        <div>
+          <p>There are around 1.8 -2 million domestic workers in Kenya</p>
```
Comment (kilemensi) [inline index.html]: Missing space before 2 million: `There are around 1.8 - 2`

File: index.html [hunk 6]
```diff
@@ -4,16 +4,25 @@
...
   <!-- Bootstrap -->
   <link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css" rel="stylesheet">
   <link href="css/bootstrap-select.min.css" rel="stylesheet">
   <link href="css/bootstrap-slider.css" rel="stylesheet">
   <link href="css/styles.css" rel="stylesheet">
+  <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> -->
 
   <!-- Font Awesome -->
-  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+  <!-- <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"> -->
```
Comment (kilemensi) [inline index.html]: Instead of commenting out, remove the stuff we no longer use.

**Review by kilemensi — APPROVED** [review]

**Comment by puritybirir** [comment]
> @puritybirir:
> 
> 1. Ensure all resources are loaded via `//domain`, you're still mixing `http://` and `https://`,
> 2. The KSH 8000 maximum seems a bit too low (both month and week will show red). But it's up to the data team I guess.
>    Screenshot from 2019-03-11 13-25-59
> 
> Other than that, I think it's good to go. 👍 .
> 
> PS: The broken Twitter and Facebook buttons was because of Firefox blocking [content](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Privacy/Tracking_Protection?utm_source=mozilla&utm_medium=firefox-console-errors&utm_campaign=default)

Fixing the resources ASAP. This is the data the team was using based on the assumptions they created

File: css/styles.css [hunk 7]
```diff
@@ -1,382 +1,561 @@
 @import url(http://fonts.googleapis.com/css?family=Roboto:400,700,900);
 @import url(http://fonts.googleapis.com/css?family=Roboto+Slab:400,700,300);
```
Comment (kilemensi) [inline css/styles.css]: Update to `https`

File: index.html [hunk 8]
```diff
@@ -4,783 +4,806 @@
...
-  <title>Living Wage | Code for South Africa </title>
+  <title>Living Wage | People Daily Kenya </title>
+  <meta property="og:url"           content="https://livingwage.codeforkenya.org/" />
+  <meta property="og:type"          content="website" />
+  <meta property="og:title"         content="Living Wage | People Daily Kenya" />
+  <meta property="og:description"   content="Are you paying your domestic worker enough?" />
+  <meta property="og:image"         content="https://livingwage.codeforkenya.org/images/womanWorking.jpg" />
 
   <!-- Bootstrap -->
   <link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css" rel="stylesheet">
```
Comment (kilemensi) [inline index.html:17]: Update to `https` (& also the link to include `bootstrap.min.js`)

**Review by kilemensi — COMMENTED** [review]
Sources missing for: **Transportation Cost**, **Communications Cost**, **Recreation Cost**, and **Other Costs** @puritybirir

**Pull request merged**